RacingCar/model/Car.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Car.swift
+//  RacingCar
+//
+//  Created by ycsong on 2022/04/28.
+//
+
+class Car {
+    public var point: Int = 0
+    
+    init() { }
+    
+    func foward() {
```

Comment:
```suggestion
    func moveFoward() {
```
라는 네이밍은 어떻게 생각하시나요? 🤔

Reply:
기존 foward 보다 직관적인 것 같습니다!